portals/devportal/source/src/app/components/Apis/Details/GraphQLConsole/GraphQLUI.jsx

```diff
@@ -80,21 +80,34 @@ export default function GraphQLUI(props) {
         setIsExplorerOpen(newExplorerIsOpen);
     };
 
+    const getURLs = () => {
+        if (api.advertiseInfo && api.advertiseInfo.advertised) {
+            return [
+                api.advertiseInfo.apiExternalProductionEndpoint,
+                api.advertiseInfo.apiExternalSandboxEndpoint,
+            ];
+        } else {
+            return URLs && URLs.https;
+        }
+    };
+
     /**
      * Execute GraphQL query
      * @param {*} graphQLParams GraphQL query parameters
      * @returns {Promise} GraphQL query promise
      */
     function graphiQLFetcher(graphQLParams) {
         let token;
-        if (authorizationHeader === 'apikey') {
+        if (api.advertiseInfo && api.advertiseInfo.advertised) {
+            token = accessTokenProvider();
+        } else if (authorizationHeader === 'apikey') {
             token = accessTokenProvider();
         } else if (securitySchemeType === 'BASIC') {
             token = 'Basic ' + accessTokenProvider();
         } else {
             token = 'Bearer ' + accessTokenProvider();
         }
-        return fetch((URLs && URLs.https), {
+        return fetch(getURLs(), {
```

Comment:
fetch expect url string as its first arg, we are passing an array of URL when api is advertised this will break the fetch

Reply:
Fixed with https://github.com/wso2/apim-apps/pull/130/commits/d1df74c14478f7df3a3afd0f7bca66d3511cba20